mutiny-core/src/nodemanager.rs

```diff
@@ -2316,27 +2316,33 @@ impl<S: MutinyStorage> NodeManager<S> {
         Ok(price)
     }
 
-    async fn fetch_bitcoin_price(&self) -> Result<f32, MutinyError> {
-        log_debug!(self.logger, "fetching new bitcoin price");
+    async fn fetch_bitcoin_price(&self, fiat: String) -> Result<f32, MutinyError> {
```

Comment:
you can make this `fiat: &str` and then you change `fiat.clone()` on line 2298 to `&fiat`